<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoodManage</class>
 <widget class="QWidget" name="GoodManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Good Manage</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Seller</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>94</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>391</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Seller</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>94</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>GoodManage</receiver>
   <slot>onAddClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>GoodManage</receiver>
   <slot>onDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>GoodManage</receiver>
   <slot>onApplyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>GoodManage</receiver>
   <slot>onResetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddClicked()</slot>
  <slot>onDeleteClicked()</slot>
  <slot>onApplyClicked()</slot>
  <slot>onResetClicked()</slot>
 </slots>
</ui>
